### PR TITLE
commit to address warnings

### PR DESCRIPTION
**Pull request by Thehashhobo**
Attempt to fix warnings in the project.

### CODE REVIEW

File: src/test/java/log_in/LogInTest.java [hunk 1]
```diff
@@ -22,7 +22,7 @@ public class LogInTest {
     LogInInputBoundary interactor;
     LogInOutputBoundary presenter;
     LogInController controller;
-    LogInUserFactory userFactory;
+    FavouriteUserFactory userFactory;
```
Comment (reganq) [inline src/test/java/log_in/LogInTest.java:25]: We'll have to fix this so it works with the LogInUserFactory instead, as that is the object being used in the log in use case

**Review by reganq — CHANGES_REQUESTED** [review]
Can you refactor the log in test so it uses the LogInUserFactory?

**Review by BerryRahul — APPROVED** [review]
good changes, looks great

**Review by izumidayo — APPROVED** [review]
Everything looks good! Thanks Jerry

**Pull request merged**